<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfDialog</class>
 <widget class="QDialog" name="ConfDialog">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Fonts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;Main Game Font</string>
           </property>
           <property name="buddy">
            <cstring>mainFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QFontComboBox" name="mainFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="mainFontSizeSpinBox">
             <property name="suffix">
              <string notr="true">pt</string>
             </property>
             <property name="minimum">
              <number>6</number>
             </property>
             <property name="maximum">
              <number>72</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;Fixed-width Font</string>
           </property>
           <property name="buddy">
            <cstring>fixedFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QFontComboBox" name="fixedFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="fontFilters">
              <set>QFontComboBox::MonospacedFonts</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="fixedFontSizeSpinBox">
             <property name="suffix">
              <string notr="true">pt</string>
             </property>
             <property name="minimum">
              <number>6</number>
             </property>
             <property name="maximum">
              <number>72</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&amp;Serifed Font</string>
           </property>
           <property name="buddy">
            <cstring>serifFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFontComboBox" name="serifFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="serifFontSizeSpinBox">
             <property name="suffix">
              <string notr="true">pt</string>
             </property>
             <property name="minimum">
              <number>6</number>
             </property>
             <property name="maximum">
              <number>72</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Sa&amp;ns-Serif Font</string>
           </property>
           <property name="buddy">
            <cstring>sansFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QFontComboBox" name="sansFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sansFontSizeSpinBox">
             <property name="suffix">
              <string notr="true">pt</string>
             </property>
             <property name="minimum">
              <number>6</number>
             </property>
             <property name="maximum">
              <number>72</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>S&amp;cript Font</string>
           </property>
           <property name="buddy">
            <cstring>scriptFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QFontComboBox" name="scriptFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="scriptFontSizeSpinBox">
             <property name="suffix">
              <string notr="true">pt</string>
             </property>
             <property name="minimum">
              <number>6</number>
             </property>
             <property name="maximum">
              <number>72</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&amp;Typewriter Font</string>
           </property>
           <property name="buddy">
            <cstring>writerFontBox</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QFontComboBox" name="writerFontBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="fontFilters">
              <set>QFontComboBox::MonospacedFonts</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="writerFontSizeSpinBox">
             <property name="suffix">
              <string notr="true">pt</string>
             </property>
             <property name="minimum">
              <number>6</number>
             </property>
             <property name="maximum">
              <number>72</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Input Font</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="useMainFontCheckBox">
            <property name="text">
             <string>Use Main Game Font for Input</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QFontComboBox" name="inputFontBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="inputFontSizeSpinBox">
            <property name="suffix">
             <string notr="true">pt</string>
            </property>
            <property name="minimum">
             <number>6</number>
            </property>
            <property name="maximum">
             <number>72</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="inputFontItalicCheckBox">
            <property name="text">
             <string>&amp;Italic</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="inputFontBoldCheckBox">
            <property name="text">
             <string>&amp;Bold</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Central Window</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="formAlignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>&amp;Text</string>
              </property>
              <property name="buddy">
               <cstring>mainTextColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="KColorButton" name="mainTextColorButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>&amp;Background</string>
              </property>
              <property name="buddy">
               <cstring>mainBgColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="KColorButton" name="mainBgColorButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>&amp;Input</string>
              </property>
              <property name="buddy">
               <cstring>inputColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="KColorButton" name="inputColorButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Banner Windows</string>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="formAlignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>T&amp;ext</string>
              </property>
              <property name="buddy">
               <cstring>bannerTextColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="KColorButton" name="bannerTextColorButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>B&amp;ackground</string>
              </property>
              <property name="buddy">
               <cstring>bannerBgColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="KColorButton" name="bannerBgColorButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="linksGroupBox">
         <property name="title">
          <string>Hyperlinks</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="underlineLinksCheckBox">
            <property name="text">
             <string>&amp;Underline Links</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="highlightLinksCheckBox">
            <property name="text">
             <string>&amp;Highlight when hovering</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <property name="formAlignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>U&amp;nvisited</string>
              </property>
              <property name="buddy">
               <cstring>linkUnvisitedColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="KColorButton" name="linkUnvisitedColorButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>H&amp;overing</string>
              </property>
              <property name="buddy">
               <cstring>linkHoveringColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="KColorButton" name="linkHoveringColorButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>&amp;Clicked</string>
              </property>
              <property name="buddy">
               <cstring>linkClickedColorButton</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="KColorButton" name="linkClickedColorButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Media</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>You can prevent games from displaying graphics or playing music and sound effects by clearing these checkboxes. If you turn off graphics and/or sounds, multimedia games will still work, but they'll be limited to displaying text.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QWidget" name="widget" native="true"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="allowGraphicsCheckBox">
             <property name="text">
              <string>Allow &amp;Graphics</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="allowSoundEffectsCheckBox">
             <property name="text">
              <string>Allow &amp;Sound Effects</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="allowMusicCheckBox">
             <property name="text">
              <string>Allow Background &amp;Music</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="allowLinksCheckBox">
             <property name="text">
              <string>Allow &amp;Hyperlinks</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true"/>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="smoothScalingCheckBox">
         <property name="text">
          <string>Use Smooth Image Scaling</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>File Safety</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>The file safety setting applies only to explicit file operations by the game. It does not affect saving or restoring games or logging a transcript to a file.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QWidget" name="widget_6" native="true"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QRadioButton" name="safety4RadioButton">
             <property name="text">
              <string>Maximum Safety - No File Access</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="safety3RadioButton">
             <property name="text">
              <string>Read-Only Access in Current Directory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="safety2RadioButton">
             <property name="text">
              <string>Read-Write Access in Current Directory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="safety1RadioButton">
             <property name="text">
              <string>Read in Any Directory, Write in Current Directory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="safety0RadioButton">
             <property name="text">
              <string>Disable Safety - Read-Write in Any Directory</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>The majority of TADS 2 games don't use Unicode to display characters not covered by the ASCII character set. They also don't tell interpreters what encoding they actually use. Here you can select the character set that should be used for TADS 2 games. Note that this setting only takes effect on newly added text; already existing text will not be converted.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>TADS 2 Encoding</string>
           </property>
           <property name="buddy">
            <cstring>encodingComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="encodingComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="softScrollCheckBox">
         <property name="text">
          <string>Soft Scrolling</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="askForGameFileCheckBox">
         <property name="text">
          <string>Ask for a game file at startup</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
